apps/web/src/data/en/index.ts

```diff
@@ -902,6 +902,7 @@ export const loggedInData = {
         exercise: {
           title: 'Exercise',
           description: 'Interactive or text based exercise',
+          placeholder: 'Type the assignment here (Optional)',
```

Comment:
@elbotho The original German text I got from Gregor was "Füge hier den Arbeitsauftrag ein. (Optional)". This was my attempt to translate it. I already asked Gregor for the English version as well, so I might update this. In any case, would you mind adding the German version to Crowdin?